api/v1.1/authors.json

```diff
@@ -3,11 +3,18 @@ layout: null
 ---
 [
   {% for author in site.author %}
+    {% if author.contracts %}
+      {% for each in author.contracts %}
+        {% assign contract = each %}
```

Comment:
`each` me semble être un très mauvais nom d'itérateur, on a l'impression qu'il s'agit d'un élément de contrôle de flux et non d'une variable. Par ailleurs, autant utiliser directement [`last`](https://shopify.github.io/liquid/filters/last/).